app/io/pathfinder/data/ObserverDao.scala

```diff
@@ -0,0 +1,49 @@
+package io.pathfinder.data
+
+import com.avaje.ebean.Model
+import com.avaje.ebean.Model.Find
+
+/**
+ * subclasses can use callbacks to listen to changes to models in the specified dao
+ * @param dao
```

Comment:
For these `@param` tags, please either remove them or add descriptions. Also, please be a little more concrete in your description of the class. I cannot explain what it means from this.